projects/1.16/assets/spheric/spheric/lang/zh_cn.json

```
@@ -0,0 +1,66 @@
+{
+  "biome.spheric.alps": "阿尔卑斯",
+  "biome.spheric.antreza": "安特雷萨",
+  "biome.spheric.dark_forest": "黑暗森林",
+  "biome.spheric.larks_forest": "云雀森林",
+  "block.spheric.antreza_button": "安特雷萨木按钮",
+  "block.spheric.antreza_chair": "安特雷萨木椅",
+  "block.spheric.antreza_door": "安特雷萨木门",
+  "block.spheric.antreza_fence_gate": "安特雷萨木栅栏门",
+  "block.spheric.antreza_fence": "安特雷萨木栅栏",
+  "block.spheric.antreza_leaves": "安特雷萨树叶",
+  "block.spheric.antreza_log": "安特雷萨原木",
+  "block.spheric.antreza_planks": "安特雷萨木板",
+  "block.spheric.antreza_pressure_plate": "安特雷萨压力板",
+  "block.spheric.antreza_sapling": "安特雷萨树苗",
+  "block.spheric.antreza_slab": "安特雷萨木台阶",
+  "block.spheric.antreza_stairs": "安特雷萨木楼梯",
+  "block.spheric.antreza_table": "安特雷萨桌子",
+  "block.spheric.antreza_trap_door": "安特雷萨活板门",
+  "block.spheric.antreza_vines": "安特雷萨藤蔓",
+  "block.spheric.ascenda_ore": "阿桑达矿石",
+  "block.spheric.lar_button": "拉尔木按钮",
+  "block.spheric.lar_chair": "拉尔木椅",
+  "block.spheric.lar_door": "拉尔木门",
+  "block.spheric.lar_fence_gate": "拉尔木栅栏门",
+  "block.spheric.lar_fence": "拉尔木栅栏",
+  "block.spheric.lar_leaves": "拉尔树叶",
+  "block.spheric.lar_log": "拉尔原木",
+  "block.spheric.lar_planks": "拉尔木板",
+  "block.spheric.lar_pressure_plate": "拉尔压力板",
+  "block.spheric.lar_slab": "拉尔木台阶",
+  "block.spheric.lar_stairs": "拉尔木楼梯",
+  "block.spheric.lar_table": "拉尔桌子",
+  "block.spheric.lar_trap_door": "拉尔活板门",
+  "block.spheric.lar_vines": "拉尔藤蔓",
+  "block.spheric.larks_button": "云雀木按钮",
+  "block.spheric.larks_chair": "云雀木椅",
+  "block.spheric.larks_door": "云雀木门",
+  "block.spheric.larks_fence_gate": "云雀木栅栏门",
+  "block.spheric.larks_fence": "云雀木栅栏",
+  "block.spheric.larks_log": "云雀原木",
+  "block.spheric.larks_planks": "云雀木板",
+  "block.spheric.larks_pressure_plate": "云雀压力板",
+  "block.spheric.larks_shroom_block": "云雀菇块",
+  "block.spheric.larks_slab": "云雀木台阶",
+  "block.spheric.larks_stairs": "云雀木楼梯",
+  "block.spheric.larks_table": "云雀桌子",
+  "block.spheric.larks_trap_door": "云雀活板门",
+  "block.spheric.larks_vera": "云雀维拉",
+  "block.spheric.larks_vines": "云雀藤蔓",
+  "block.spheric.larkser_shroom": "云雀蘑菇",
```

```suggestion
  "block.spheric.larks_button": "拉克斯木按钮",
  "block.spheric.larks_chair": "拉克斯木椅",
  "block.spheric.larks_door": "拉克斯木门",
  "block.spheric.larks_fence_gate": "拉克斯木栅栏门",
  "block.spheric.larks_fence": "拉克斯木栅栏",
  "block.spheric.larks_log": "拉克斯原木",
  "block.spheric.larks_planks": "拉克斯木板",
  "block.spheric.larks_pressure_plate": "拉克斯木压力板",
  "block.spheric.larks_shroom_block": "拉克斯菇方块",
  "block.spheric.larks_slab": "拉克斯木台阶",
  "block.spheric.larks_stairs": "拉克斯木楼梯",
  "block.spheric.larks_table": "拉克斯木桌",
  "block.spheric.larks_trap_door": "拉克斯木活板门",
  "block.spheric.larks_vera": "拉克斯叶簇",
  "block.spheric.larks_vines": "拉克斯藤蔓",
  "block.spheric.larkser_shroom": "拉克斯菇",
```
`vera`疑似来自`Aloe vera`，aloe单独也用于称呼芦荟
可能是错将种加词vera（意为“真”）当做是叶簇的意思了吧（）
可根据实际形象改译